kolibri/core/negotiation.py

```diff
@@ -1,21 +1,15 @@
-from rest_framework.negotiation import BaseContentNegotiation
-from rest_framework.parsers import JSONParser
-from rest_framework.parsers import MultiPartParser
+from rest_framework.negotiation import DefaultContentNegotiation
 from rest_framework.renderers import JSONRenderer
 
 
-class LimitContentNegotiation(BaseContentNegotiation):
-    def select_parser(self, request, parsers):
+class LimitContentNegotiation(DefaultContentNegotiation):
+    def select_renderer(self, request, renderers, format_suffix=None):
         """
-        Always return JSONParser unless a 'multipart/form-data' content type is sent
-        """
-        if "multipart" in request.content_type:
-            return MultiPartParser()
-        return JSONParser()
-
-    def select_renderer(self, request, renderers, format_suffix):
-        """
-        Always return JSONRenderer
+        Always return JSONRenderer unless for morango
         """
+        if request.path.startswith("/api/morango"):
```

Comment:
Is there a more robust way to get this URL? Maybe using a `reverse` of some sort? There's a chance Kolibri has been mounted at a different subpath (@rtibbles knows about that process) -- though I'm not sure whether Morango actually supports being mounted like that (whether peers will then discover it correctly, etc).

Reply:
Ah, I didn't know that was a known use case. Okay, I think we could just remove this. I was getting another JSON decode error which I can no longer reproduce.